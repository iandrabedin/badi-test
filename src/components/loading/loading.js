import React from "react";
import "./loading.scss";

export const Loading = () => {
  return <div className="loading">Loading...</div>;
};
