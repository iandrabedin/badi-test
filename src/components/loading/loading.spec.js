import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Loading } from "./loading";

afterEach(cleanup);

describe("Loading", () => {
  it("should render as expected", () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("should have loading style", () => {
    const { getByText } = render(<Loading />);
    const loading = getByText("Loading...");
    expect(loading).toHaveClass("loading");
  });
});
