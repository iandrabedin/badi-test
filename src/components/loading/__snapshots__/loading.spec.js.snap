// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render as expected 1`] = `
<DocumentFragment>
  <div
    class="loading"
  >
    Loading...
  </div>
</DocumentFragment>
`;
